gatling-http/src/main/scala/io/gatling/http/action/ws/fsm/WsIdleState.scala

```diff
@@ -109,8 +109,13 @@ final class WsIdleState(fsm: WsFsm, session: Session, webSocket: WebSocket) exte
   }
 
   override def onTextFrameReceived(message: String, timestamp: Long): NextWsState = {
-    // server push message, just log
-    logUnmatchedServerMessage(session)
+    // try to auto reply or log the message
+    if (fsm.httpProtocol.wsPart.autoReplyTextFrames.isDefinedAt(message)) {
```

Comment:
Instead of duplicating this 3 times, it would be better to make it a protected method in parent `WsState`.
Also, we probably want to create a direct reference to autoReplyTextFrames instead of traversing it once, and possibly twice every time.

Reply:
done